books/free-programming-books-de.md

```diff
@@ -188,7 +188,7 @@
 
 #### Symfony
 
-* [Symfony: Auf der Überholspur](https://symfony.com/doc/current/the-fast-track/de/index.html) (Online)
+* [Symfony: Auf der Überholspur](https://symfony.com/doc/5.4/the-fast-track/en/index.html) (Online)
```

Comment:
Looks like you have linked a English version of the book, whereas this file is for german